### PR TITLE
feat(curriculum): add sentence maker lab

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by moT01 — APPROVED** [review]
LGTM 🎉

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1882,6 +1882,7 @@
       "hoec": { "title": "136", "intro": [] },
       "cygu": { "title": "137", "intro": [] },
       "axgb": { "title": "138", "intro": [] },
+      "nnqa": { "title": "154", "intro": [] },
```
Comment (jdwilkin4) [inline client/i18n/locales/english/intro.json:1885]: this could be deleted right?

**Review by jdwilkin4 — APPROVED** [review]
LGTM 👍 

I left one comment regarding the intro.json

File: client/i18n/locales/english/intro.json (hunk 1 above)
Comment (zairahira) [reply client/i18n/locales/english/intro.json]: Yes, this shouldn't be here. I have removed it.

**Review by Sembauke — APPROVED** [review]

**Pull request merged**